### PR TITLE
Online

### PR DESCRIPTION
**Pull request by benfoley**
These updates replaces seperate transcribe and transcribe-align flow with a single method using Nick's gmm-decode-long script. Minor bug found in the gmm-decode-long script which is from wav.scp needing two different formats.

### CODE REVIEW

File: elpis/wrappers/inference/gmm-decode-long.sh [hunk 1]
```diff
@@ -34,16 +34,92 @@ echo "# dummy file" > ./conf/online_cmvn.conf
...
+line=$(head -n 1 ./data/infer/spk2utt)
+utt=` echo ${line} | cut -d ' ' -f 2`
+spk=` echo ${line} | cut -d ' ' -f 1` # this was seg
+audio="audio.wav"
+length=`sox --i -D ${audio}`
+recid="decode"
+
+# Prepare the split dir
+splitDir=./data/infer/split1/
+if [ -d $splitDir ]; then rm -r $splitDir; fi
```
Comment (nicklambourne) [inline elpis/wrappers/inference/gmm-decode-long.sh]: As I've learnt from my extensive immersion in bash these holidays, the `[[ -d $splitDir ]]` format is safer to use with modern bash.

Comment (benfoley) [reply elpis/wrappers/inference/gmm-decode-long.sh]: ok, fixed in 654dbf2

File: Dockerfile [hunk 2]
```diff
@@ -121,7 +121,7 @@ ADD http://www.random.org/strings/?num=10&len=8&digits=on&upperalpha=on&loweralp
 
 # Elpis
 WORKDIR /
-RUN git clone --depth=1 https://github.com/CoEDL/elpis.git
+RUN git clone -b online --depth=1 https://github.com/CoEDL/elpis.git
```
[the remark this answers is not in the document]
Comment (benfoley) [reply Dockerfile]: ah yes, thanks. fixed in 94230fc

**Comment by alexis-michaud** [comment]
👍 👍

**Pull request merged**